### PR TITLE
Bjorncs/jetty graceful shutdown

### PR DESCRIPTION
**Pull request by bjorncs**
FYI @hmusum @gjoranv

### CODE REVIEW

File: jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java [hunk 1]
```diff
@@ -307,7 +307,7 @@ public void start() {
     @Override
     public void close() {
         try {
-            log.log(Level.INFO, String.format("Shutting down server (graceful=%b)", isGracefulShutdownEnabled()));
+            log.log(Level.INFO, String.format("Shutting down server (graceful=%b, timeout=%.1fs)", isGracefulShutdownEnabled(), server.getStopTimeout()/1000d));
```
Comment (gjoranv) [inline jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java]: If the timeout is irrelevant when graceful=false, it should not be printed.

**Review by gjoranv — COMMENTED** [review]
🥇 for removing guava usage

**Review by baldersheim — APPROVED** [review]
:+

File: jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java (hunk 1 above)
Comment (bjorncs) [reply jdisc_http_service/src/main/java/com/yahoo/jdisc/http/server/jetty/JettyHttpServer.java]: The timeout still impacts other parts of Jetty (stopTimeout is generic concept for all `LifeCycle` objects in Jetty).

**Pull request merged**